helpdesk_mgmt_sla/views/helpdesk_sla_views.xml

```diff
@@ -14,17 +14,24 @@
                     <group>
                         <group>
                             <separator colspan="2" string="Apply on" />
+                            <field name="team_ids" widget="many2many_tags" />
+                            <field name="category_ids" widget="many2many_tags" />
+                            <field name="tag_ids" widget="many2many_tags" />
                             <field
-                                name="team_ids"
-                                domain="[('use_sla', '=', True)]"
-                                widget="many2many_tags"
+                                name="domain"
+                                widget="domain"
+                                options="{'model': 'helpdesk.sla', 'in_dialog': True}"
                             />
                             <separator colspan="2" string="Target" class="mt32" />
-                            <field name="stage_id" string="Reach Stage" />
+                            <field name="stage_ids" string="Reach Stages" />
```

Comment:
```suggestion
                            <field name="stage_ids"  />
```
And move it before `<field name="domain" />`, because it's part of the filters